tests/block-period-api.test.js

```
@@ -0,0 +1,321 @@
+    await assert.rejects(
+      () =>
+        BlockPeriodService.getAvailableBlockPeriods(
+          "tenant-1",
+          "camping-a",
+          "2026-01-01",
+          "2026-12-31",
+          1,
+          null,
+        ),
+      (error) =>
+        error instanceof BadRequestError && error.code === "date_range_too_large",
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)